moban/engine.py

```diff
@@ -96,6 +98,8 @@ def _file_permissions_copy(self, template_file, output_file):
         utils.file_permissions_copy(true_template_file, output_file)
 
     def _apply_template(self, template, data, output):
+        self.jj2_environment.globals['__target__'] = output
+        self.jj2_environment.globals['__template__'] = template.name
```

Comment:
There are two ways to solve this:-
1.  Add the new variables to `data` before the render call so that it will be specific to a template.
2.  `Template.globals` . The docs say `Template.globals exist that are variables available to a specific template that are available to all render() calls.`